MAUI/Parallax-View/migration.md

```diff
@@ -0,0 +1,202 @@
+
+<table>
+<tr>
+<th>Xamarin</th>
+</tr>
+<tr>
+<td>
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+    <ContentPage
+        . . .
+        xmlns:parallax="clr-namespace:Syncfusion.XForms.ParallaxView;assembly=Syncfusion.XForms.ParallaxView">
+
+        <parallax:SfParallaxView/>
+
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+    using Syncfusion.XForms.ParallaxView;
+    ...
+
+    SfParallaxView parallax = new SfParallaxView();
+    this.Content = parallax;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+<tr>
+<th>.NET MAUI</th>
+</tr>
+<tr>
+<td>
+{% tabs %} 
+
+{% highlight xaml %}
+
+    <ContentPage
+        . . .    
+        xmlns:parallax="clr-namespace:Syncfusion.Maui.ParallaxView;assembly=Syncfusion.Maui.ParallaxView">
+
+        <parallax:SfParallaxView/>
+
+    </ContentPage>
+ 
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.ParallaxView;
+. . .
+SfParallaxView parallax = new SfParallaxView();
+this.Content = parallax;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## Classes
+
+<table>
+<tr>
+<th>Xamarin</th>
+<th>.NET MAUI</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>SfParallaxView</td>
+<td>SfParallaxView</td>
+<td>SfParallaxView control provides scrolling for any control that implements the IParallaxView.</td>
+</tr>
+<tr>
+<td>ParallaxScrollingEventArgs</td>
+<td>ParallaxScrollingEventArgs</td>
+<td>Defines the scrolling Event args</td>
+</tr>
+</table>
+
+## Interface
+
+<table>
+<tr>
+<th>Xamarin</th>
+<th>.NET MAUI</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>IParallaxView</td>
+<td>IParallaxView</td>
+<td>Defines a event and a property to perform parallax scrolling.</td>
+</table>
+
+## Enums
+
+<table>
+<tr>
+<th>Xamarin</th>
+<th>.NET MAUI</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>Orientation</td>
+<td>Orientation</td>
+<td>Defines the orientation of the view</td>
+</tr>
+</table>
+
+## API Migration
+The following table illustrates the API migration for the parallax view.
+
+### Parallax View
+
+<table>
+<tr>
+<th>Xamarin</th>
+<th>.NET MAUI</th>
+</tr>
+<tr>
+<td>Content</td>
+<td>Content</td>
+</tr>
+<tr>
+<td>Source</td>
+<td>Source</td>
+</tr>
+<tr>
+<td>Speed</td>
+<td>Speed</td>
+</tr>
+<tr>
+<td>Orientation</td>
+<td>Orientation</td>
+</tr>
+<tr>
+<td>Scrolling</td>
+<td>Scrolling</td>
+</tr>
+<tr>
+<td>ScrollX</td>
+<td>ScrollX</td>
+</tr>
+<tr>
+<td>ScrollY</td>
+<td>ScrollY</td>
+</tr>
+<tr>
+<td>CanAnimate</td>
+<td>CanAnimate</td>
+</tr>
+<tr>
+<td>ScrollContentSize</td>
+<td>ScrollContentSize</td>
+</tr>
+</table>
+
+## Support and feedback
+
+If you are unable to find the migration information you require in the self-help resources listed above, please contact us by creating a [support ticket](https://internalsupport.bolddesk.com/agent/tickets/create). If you do not see what you need, Please request it in our [feedback portal](https://www.syncfusion.com/feedback/maui). 
```

Comment:
If you are unable to find the migration information you require in the self-help resources listed above, please contact us by creating a [support ticket](https://internalsupport.bolddesk.com/agent/tickets/create). If you don't find what you need, please request it from our [feedback portal](https://www.syncfusion.com/feedback/maui).